lookout/style/format/benchmarks/expected_vnodes_number.py

```diff
@@ -0,0 +1,142 @@
+"""Util to workaround https://github.com/src-d/style-analyzer/issues/557."""
+import csv
+import functools
+import logging
+import logging.handlers
+import os
+import subprocess
+import sys
+import tempfile
+from typing import Optional
+
+from lookout.core.test_helpers import server
+from tqdm import tqdm
+
+from lookout.style.format.analyzer import FormatAnalyzer
+from lookout.style.format.benchmarks.top_repos_quality import AnalyzerContextManager, \
+    ensure_repo, handle_input_arg
+from lookout.style.format.feature_extractor import FeatureExtractor
+
+analyzer_class = FormatAnalyzer
+
+
+def _stop_bblfshd():
+    subprocess.check_call("docker rm -f expected_vnodes_bblfshd".split(),
+                          stdout=sys.stderr, stderr=sys.stderr)
+
+
+def _restart_bblfshd(first: bool=False) -> str:
+    log = logging.getLogger("expected_vnodes_number")
+    port = server.find_port()
+    log.info("Restarting bblfshd")
+    if not first:
+        _stop_bblfshd()
+    subprocess.check_call(("docker run -d --rm --name expected_vnodes_bblfshd --privileged -p "
```

Comment:
We already have `docker` Python package in our deps. It can do the same - can you please change subprocess calls to API calls